src/main/java/seedu/module/model/tag/Tag.java

```diff
@@ -19,7 +19,7 @@
      *
      * @param tagName A valid tag name.
      */
-    public Tag(String tagName) {
+    public Tag (String tagName) {
```

Comment:
Err..is this space suppose to be there?

Reply:
oops

Reply:
I thought the initial checkstyle rules stated that a space is needed before the brackets